pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -530,9 +528,7 @@ func (asw *actualStateOfWorld) MarkVolumeAsResized(
 			podName,
 			volumeName)
 	}
-
-	klog.V(5).Infof("Volume %s(OuterVolumeSpecName %s) of pod %s has been resized",
-		volumeName, podObj.outerVolumeSpecName, podName)
+	klog.V(5).InfoS("Pod volume has been resized", "podName", podName, "volumeName", volumeName, "outerVolumeSpecName", podObj.outerVolumeSpecName)
```

Comment:
I'm worried that the pod name passed here `podName` is not the type we usually want to put under this key. Please look at the type `volumetypes.UniquePodName`. Please either:
* [Preferred] Try to extract normal pod name of k8s object and use `podName` key.
* Use different key like `uniquePodName` to differentiate.

Reply:
Yes,you are right.
I have fixed all of them in this file according to "Use different key like uniquePodName to differentiate.".